src/drive/web/modules/upload/index.js

```diff
@@ -138,30 +176,55 @@ const uploadFile = async (client, file, dirID) => {
   return resp.data
 }
 
+export const getFilePath = async (client, file, dirID) => {
+  const resp = await client.collection('io.cozy.files').get(dirID)
+  const parentDirectory = resp.data
+  return `${parentDirectory.path}/${file.name}`
+}
+
+export const overwriteFile = async (client, file, path) => {
+  const statResp = await client.collection('io.cozy.files').statByPath(path)
```

Comment:
Is there any reason to use statByPAth() instead of a simple `get`? Since `get` returns fullpath it can be easier to deal with?

Reply:
I don't think I can use `get` to retrieve file id and dir id from the file's path

Reply:
I think jsdoc will be useful here. I expected that `file` was a file object.